src/cdev/commands/sync.py

```diff
@@ -1,19 +1,28 @@
 from cdev.constructs.project import Project
-from cdev.default.output_manager import CdevOutputManager
-from cdev.cli.logger import set_global_logger_from_cli
 
 from core.commands.sync import core_sync_command
+from core.constructs.output_manager import OutputManager
 
 
-def sync_command_cli(args) -> None:
-    config = args[0]
-    set_global_logger_from_cli(config.loglevel)
-    sync_command(args)
+def sync_command_cli(
+    disable_prompt: bool,
+    no_default: bool,
+    watch: str,
+    ignore: str,
+    project: Project,
+    output_manager: OutputManager,
+    **kwargs
+) -> None:
+    sync_command(disable_prompt, no_default, watch, ignore, project, output_manager)
 
 
-def sync_command(args) -> None:
-
-    output_manager = CdevOutputManager()
-    my_project = Project.instance()
-    ws = my_project.get_current_environment().get_workspace()
-    core_sync_command(ws, output_manager, args)
+def sync_command(
+    disable_prompt: bool,
+    no_default: bool,
+    watch: str,
+    ignore: str,
+    project: Project,
+    output_manager: OutputManager,
+) -> None:
+    ws = project.get_current_environment().get_workspace()
```

Comment:
We use this function quite often to get the current workspace maybe a util function:
```
project.get_current_environment_workspace()
```

Reply:
Will consider. Probably out of scope for trying to get this PR merged.